ferry/database/models.py

```diff
@@ -379,6 +375,10 @@ class Listing(BaseModel):
     )
     course = relationship("Course", backref="listings_staged", cascade="all")
 
+    school = Column(
+        String, comment="School (e.g. YC, GS, MG) that the course is listed under"
+    )
+
```

Comment:
Do we need to do anything to set the field on the listings table instead of the courses table?

Reply:
Nope - in `transform.py`, the listings table and the courses table have the same columns (save for primary keys) up until the end, where their columns are selected according to the model. As a result, this is all we need to move school into the listings table. (I also re-ran the whole workflow to test and it checks out).

Reply:
Whoa that's kinda neat